PowerShellTools/ServiceManagement/PowershellHostProcessHelper.cs

```diff
@@ -27,33 +29,76 @@ internal static class PowershellHostProcessHelper
         private const uint TOPMOST_FLAGS = SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE;
         private const int SW_HIDE = 0;
 
-        public static PowershellHostProcess CreatePowershellHostProcess()
+        private static readonly ILog Log = LogManager.GetLogger(typeof(PowershellHostProcessHelper));
+

```

Comment:
Do these need to be static as part of the Helper class? Can these be encapsulated within the PowerShellHostProcess?